exercises/run-length-encoding/example.jl

```diff
@@ -0,0 +1,26 @@
+function encode(s)
+    (length(s) == 0) && (return s)
+
+    count = 0
+    for char in s
+        if char == s[1]
```

Comment:
This could also use the ternary operator.

Reply:
Wow thanks for being detailed with the check! =)

This one I intended for the `break` to be visible and thus settled on this structure, but I was perplexed as I spent quite a while thinking about which is better, without a clear winner for me in the end. Ternary with control flow like `break` will make the code terse and appeal to stronger coders but might be off-putting to beginners who will find it too hard to read, kinda like how Haskell is...

I think I can respect your opinion on this. Will edit.

Reply:
Imo the syntax highlighter will make it visible enough but I see your point. In the end it's mostly personal preference.